content/references/protocol-reference/transactions/transaction-metadata.md

```diff
@@ -68,6 +68,15 @@ A `ModifiedNode` object contains the following fields:
 
 **Note:** If the modified ledger object has `PreviousTxnID` and `PreviousTxnLgrSeq` fields, the transaction always updates them with the transaction's own identifying hash and the index of the ledger version that included the transaction, but these fields' new value is not listed in the `FinalFields` of the `ModifiedNode` object, and their previous values are listed at the top level of the `ModifiedNode` object rather than in the nested `PreviousFields` object.
 
+## NFT Fields
+
+Transactions involving NFTs can contain the following fields in the metadata. These values are added by the Clio server at request time and are not stored in the hashed binary  metadata:
```

Comment:
should we be explicit that these new fields are for responses `tx` and `account_tx` APIs only? People may be confused after they submit a transaction and don't see the field.

Reply:
Will they be present on `nft_history` too?

Reply:
@ckniffen no lol. It's only for rippled. i def forgot about it, it was a while ago. But i think i will create a ticket for it